projects/log4j2/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://logging.apache.org/log4j/2.x/"
+language: jvm
+primary_contact: "?"
```

Comment:
Please just move one of your email to primary contact for you. We plan to start conversations with log4j team soon.